ParcelDistributionCenter/ParcelDistributionCenter.Concole/Program.cs

```diff
@@ -11,7 +11,7 @@ static void Main(string[] args)
 
             // string ParcelUser = File.ReadAllText(@"D:\packages.json");//odczytanie json z pliku. Ścieżkę dodana na sztywno.
             //plik Json umieszczony w projekcie :jcszr8-DeBurgerzy\ParcelDistributionCenter\ParcelDistributionCenter.Concole\bin\Debug\net6.0
-            string ParcelUser = File.ReadAllText("packages.json");
+            string ParcelUser = File.ReadAllText("..\\..\\..\\..\\ParcelDistributionCenter.Model\\json\\packages.json");
```

Comment:
Zmieniłem lokalizacje jsona i dzięki temu nie jest ignorowany przez gita. 
Umieściłem go w Modelu bo są to statyczne dane. Jest też możliwość łatwego podglądu pliku z VS.
Pytanie jaka jest dobra praktyka w lokowaniu takich plików?